/* BeginDocumentation
Name: testsuite::test_fail - test if SLI returns exit code 126 when executing a file with unhandled errors.

Synopsis: (test_fail) run -> 

Description:
This script provokes an error that is not handled by any user error
handler. It is used to test if SLI quits with exit code 126 after
executing a file that contains errors.

Author: Ruediger Kupper

FirstVersion: 03-jul-2008

Remarks:
Yes, this script is really meant to fail!

SeeAlso: testsuite::test_pass, testsuite::test_fail, testsuite::test_stop, testsuite::test_goodhandler,  testsuite::test_lazyhandler, testsuite::test_badhandler
*/

% Yes, this script fails. This is intentional. Really.
% And no, there shall be no stopped context around this error. Really. It simply fails. That's all.

%provoke an error:
1 0 div
